### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by paolosalvatori**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* First Commit

### CODE REVIEW

File: demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,1941 @@
...
+    },
+    "aksClusterKubernetesVersion": {
+      "type": "string",
+      "defaultValue": "1.21.2",
+      "metadata": {
+        "description": "Specifies the version of Kubernetes specified when creating the managed cluster."
+      }
+    },
+    "aksClusterAdminUsername": {
+      "defaultValue": "azureuser",
```
Comment (bmoore-msft) [inline demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json:127]: remove defaultValue - increases surface area

File: demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,1941 @@
...
+    "vmAdminPasswordOrKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Specifies the SSH Key or password for the virtual machine. SSH key is recommended."
+      }
+    },
+    "diskStorageAccounType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
```
Comment (bmoore-msft) [inline demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json:709]: os disk doesn't support all storageTypes - need to remove those that are not supported

File: demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,1941 @@
...
+    "bastionPublicIpAddressId": "[resourceId('Microsoft.Network/publicIPAddresses', variables('bastionPublicIpAddressName'))]",
+    "bastionSubnetName": "AzureBastionSubnet",
+    "bastionSubnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('virtualNetworkName'), variables('bastionSubnetName'))]",
+    "workspaceId": "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('logAnalyticsWorkspaceName'))]",
+    "contributorRoleId": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', variables('contributorRoleDefinitionName'))]",
+    "contributorRoleAssignmentName": "[guid(concat(resourceGroup().id, variables('aksClusterUserDefinedManagedIdentityName'), parameters('aksClusterName')))]",
+    "contributorRoleAssignmentId": "[resourceId('Microsoft.Authorization/roleAssignments', variables('contributorRoleAssignmentName'))]",
+    "acrPullRoleId": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', variables('acrPullRoleDefinitionName'))]",
+    "acrPullRoleAssignmentName": "[guid(concat(resourceGroup().id, 'acrPullRoleAssignment'))]",
```
Comment (bmoore-msft) [inline demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json:882]: this needs to include principal, role and scope for the seed

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json (hunk 1 above)
Comment (paolosalvatori) [reply demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json:127]: I always forget to remove default values for parameters in the template that have a value in the parameters file 😃  Fixed

File: demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json (hunk 2 above)
Comment (paolosalvatori) [reply demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json:709]: Same as above. This time I also changed this in my local templates, hopefully next time I'll use the right allowed values 😄 Fiex

File: demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json (hunk 3 above)
Comment (paolosalvatori) [reply demos/private-aks-cluster-with-public-dns-zone/azuredeploy.json:882]: Check, I think it's fine now!

**Pull request merged**